### ISSUE TITLE
Cross_Entropy Loss Function in Logistic Regression

### ISSUE THREAD

**Issue by LiZongyue** [aymericdamien/TensorFlow-Examples#329]
You've defined it as 
cost = tf.reduce_mean(-tf.reduce_sum(y * tf.log(pred), axis = 1))

But I think it should be 
cost = tf.reduce_mean(-tf.reduce_sum(y * tf.log(pred), axis = 0))

Because the mean should be calculated at the end as you implemented.
